tests/test_memory.py

```diff
@@ -72,7 +73,8 @@ def __init__(self):
 
         # Create a memory gateway
         self.ms = rogue.interfaces.memory.TcpServer("127.0.0.1",9080);
-        pr.busConnect(self.ms,self.sim)
+        #self.ms >> self.sim
```

Comment:
Do you want to remove the commented code?   Or add another part in the test code that test `>>` (other direction) of the bus connecting?